.travis.yml

```diff
@@ -139,6 +139,7 @@ matrix:
         - ./utils/ci/install_cargo_web.sh
         - cargo web prepare-emscripten
         - cargo web -V
+        - cargo install wasm-pack
```

Comment:
We cache the `.cargo` directory so this may not work on the *next* build. The simple fix is to use `--force`, but this requires building the entire thing every time.

In the past we've also tried installing only if necessary like this, though it can cause problems since it won't ever upgrade:
```
cargo --list | egrep "^\s*web$" -q || cargo install cargo-web
```
Best would probably be to go back to using `cargo-update` (which I believe we stopped using due to a Cargo bug); so use the following to install wasm-pack:
```
cargo list|grep install-update || cargo install -f cargo-update
cargo install-update -i cargo-update wasm-pack
```